plugins/woocommerce/includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -532,16 +532,16 @@ public function check_and_hold_coupon( $coupon ) {
 		);
 
 		$query_for_tentative_usages = $this->get_tentative_usage_query( $coupon->get_id() );
-		$db_timestamp               = $wpdb->get_var( 'SELECT UNIX_TIMESTAMP() FROM DUAL' );
+		$db_timestamp               = $wpdb->get_var( 'SELECT UNIX_TIMESTAMP() FROM ' . $wpdb->posts . ' LIMIT 1' );
```

Comment:
A little curious if we could ditch the table reference completely?

```suggestion
		$db_timestamp               = $wpdb->get_var( 'SELECT UNIX_TIMESTAMP()' );
```

Reply:
This might be a valid SQL statement, however it won't address the issue I'm working with. In this hosting environment, the query is parsed and the table name is used to determine which servers should be used for the query. To satisfy this requirement, we would need to use a table name, and that table name needs to be one the host recognizes.